gst/nnstreamer/elements/gsttensor_debug.c

```diff
@@ -0,0 +1,441 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/**
+ * GStreamer/NNStreamer tensor_debug
+ * Copyright (C) 2022 MyungJoo Ham <myungjoo.ham@samsung.com>
+ */
+/**
+ * @file	gsttensor_debug.c
+ * @date	23 Sep 2022
+ * @brief	GStreamer plugin to help debug tensor streams.
+ *
+ * @see		https://github.com/nnstreamer/nnstreamer
+ * @author	MyungJoo Ham <myungjoo.ham@samsung.com>
+ * @bug		No known bugs except for NYI items
+ */
+
+/**
+ * SECTION:element-tensor_debug
+ *
+ * A filter that generates debug messages for developer at the insertion
+ * point of the given pipeline. An application writer using an nnstreamer
+ * pipeline can use tensor_debug to debug or get profile information in their
+ * applications.
+ *
+ * Note that this does not support other/tensor, but only supports other/tensors.
+ *
+ * <refsect2>
+ * <title>Example launch line</title>
+ * |[
+ * gst-launch-1.0 videotestsrc ! video/x-raw,format=RGB,width=640,height=480 ! tensor_converter ! tensor_debug output=CONSOLE_E cap=SHOW_ALWAYS ! tensor_sink
```

Comment:
```
gst-launch-1.0 videotestsrc ! video/x-raw,format=RGB,width=640,height=480 ! tensor_converter ! tensor_debug output-methods=console-error capability=always ! tensor_sink
```
How about changing output to output-methods and option values?
CONSOLE_E, SHOW_ALWAYS make erroneous pipline warning.

Reply:
Fixed! Thanks!